spec/features/user_views_genre_page_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe 'User views genre page' do
+  it 'should see books related to this genre' do
+    book1 = create :book
+    book2 = create :book
+    genre = create :genre, books: [book1, book2]
```

Comment:
@Fatpaher You can use `create_list` and pass resulting array to `create :genre`. Also it will allow you to dry expectations in this spec.